Aws data lake fix yml (#32870)
demisto/content: Packs/AWS-SecurityLake/ReleaseNotes/1_0_3.md

#### Integrations

##### AWS-SecurityLake

> - Fixed an issue where teh context data in the yml and in the documentation was wrong.

Reviewer: ```suggestion
- Fixed an issue where the context path of command outputs was incorrect.
```
